src/gluonts/transform/feature.py

```diff
@@ -45,6 +45,12 @@ class AddObservedValuesIndicator(SimpleTransformation):
         Field name to use for the indicator
     dummy_value
         Value to use for replacing missing values.
+    imputation_method
+        Select the method to replace the missing values.
+        - "standard" to just replace them with the dummy_value
```

Comment:
Maybe “dummy_value” is a more appropriate name for this option?

Reply:
Fair enough, I can change this to "dummy_value"